pyscf/lib/linalg_helper.py

```diff
@@ -31,9 +31,9 @@
 from pyscf.lib import misc
 from pyscf import __config__
 
-SAFE_EIGH_LINDEP = getattr(__config__, 'lib_linalg_helper_safe_eigh_lindep', 1e-15)
+SAFE_EIGH_LINDEP = getattr(__config__, 'lib_linalg_helper_safe_eigh_lindep', 1e-13)
```

Comment:
This setting affects the mcscf convergence. Please change it back to 1e-15 for backward reproduciblity .